src/libs/migrations/PersonalDetailsByAccountID.js

```diff
@@ -251,12 +251,6 @@ export default function () {
                     delete newReport.lastActorEmail;
                 }
 
-                if (lodashHas(newReport, ['participants'])) {
-                    reportWasModified = true;
-                    Log.info(`[Migrate Onyx] PersonalDetailsByAccountID migration: removing participants from report ${newReport.reportID}`);
-                    delete newReport.participants;
-                }
-
```

Comment:
Why removing this block in particular?

Reply:
If we don't remove this then when we re-introduce the `report.participants` data this old Onyx migration will delete it.

We could avoid it by using a different name - but this is a pretty old migration and we will force users to update the app so it's not really needed anymore.